src/components/IstioWizards/RequestRouting/Rules.tsx

```diff
@@ -137,9 +138,7 @@ class Rules extends React.Component<Props> {
                       .map((wk, i) => {
                         return (
                           <div key={'wk_' + order + '_' + wk.name + '_' + i}>
-                            <Tooltip position={TooltipPosition.top} content={<>Workload</>}>
-                              <Badge className={'virtualitem_badge_definition'}>WS</Badge>
-                            </Tooltip>
+                            {pfBadge(PFBadges.Workload, TooltipPosition.top)}
```

Comment:
I don't have a great answer.  I think it just sort of came out this way.  But, yeah, maybe it would be more natural.  I'll sleep on it and let you know tomorrow,

Reply:
I didn't sleep, I changed it, please take a look at the update.